### ISSUE TITLE
[Regression] Driver: San Francisco [BLES00891] freezes after a short period of time driving from 0.0.23-14079 onwards

### ISSUE THREAD

**Issue by solarmystic** [RPCS3/rpcs3#12640]
### Quick summary

The game, which has been reported as Playable for the past 2 years or so, freezes after a short period of time driving since 0.0.23-14079 #12523 onwards.

EDITED for corrections.

### Details

~~Even though 14080 is the first build to cause the freeze, this game happens to be a part of a small group of games that underwent heavy performance regression by the build that immediately preceded it (0.0.23-14079), which has been reported in #12541.~~

**(Correction:- 14079 does freeze after a longer while in game, but I suspect it takes longer to trigger it since the performance is so atrocious)**

The last truly non-problematic build for the game is 0.0.23-14068 #12538 

**Short regression history:-**

0.0.23-14068 - Playable as per compatibility listing.
0.0.23-14079 - Severe performance regression (as reported in #12541) and freezes after driving for a while.
0.0.23-14080 - Performance regression fixed. However, the freezing is still present until the current master.

### Build with regression
v0.0.23-14079-bf63a18c

### Attach two log files

Working
[RPCS3 14068 okay.log.gz](https://github.com/RPCS3/rpcs3/files/9542614/RPCS3.14068.okay.log.gz)

Not working 
[RPCS3 14141.log.gz](https://github.com/RPCS3/rpcs3/files/9542612/RPCS3.14141.log.gz)

### System configuration

SYS: RPCS3 v0.0.23-14068-a74d3005 Alpha | master
SYS: 12th Gen Intel(R) Core(TM) i5-12400F | 12 Threads | 31.86 GiB RAM | TSC: 2.496GHz | AVX+ | FMA3
SYS: Operating system: Windows, Major: 10, Minor: 0, Build: 19043, Service Pack: none, Compatibility mode: 0
RSX: Found vulkan-compatible GPU: 'NVIDIA GeForce RTX 2080 Ti' running on driver 516.94.0.0
SYS: Firmware version: 4.88

**Comment by Jonathan44062** [comment]
I can confirm, the build 0.0.23-14079 affect Sonic Unleashed, it freezes after after a short period of time and still does in recent builds 
In the build 0.0.23-14080, it freezes
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9628729/RPCS3.log.gz)

**Comment by SimplyTadpole** [comment]
I can also attest the latest build of RPCS3 causes Driver: San Francisco to freeze after about a minute or two of in-game gameplay (it does not seem to affect the main menu). The only other PS3 game I run RPCS3 in, MotorStorm: Apocalypse, does not have the freezing issue.

**Attach log file**
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9790859/RPCS3.log.gz)

**Comment by Triticum0** [comment]
Had the same issue but it freezes while in the cutscene

**Comment by VIPLightning** [comment]
Have you tried changing the thread scheduler to RPCS3 scheduler instead of the OS? It seems like it has improved the performance and less crashes on the latest build. I managed to get through a few missions.

**Comment by Ordinary205** [comment]
The regression seems to be fixed after the 0.0.28-15118 version
I have not encountered any freezes while I was testing this game.
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11687546/RPCS3.log.gz)

**Comment by solarmystic** [comment]
> The regression seems to be fixed after the 0.0.28-15118 version I have not encountered any freezes while I was testing this game. But I did enabled Atomic for better stability. [RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11687546/RPCS3.log.gz)

Confirmed, ran the game for an hour with no freezing on latest master. 
Atomic FIFO is mandatory though, it still freezes within minutes if FIFO is set to Fast.
Prior to this issue, on builds 14068 and older, the game was considered Playable with just Fast FIFO.
I will close this since this is fixed, but the rpcs3 wiki will need to be updated with this new stability requirement.

**Comment by solarmystic** [comment]
Re-opening since a user from discord can still cause it freeze on latest build even with Atomic FIFO after 30 minutes of gameplay.
https://discord.com/channels/272035812277878785/272035812277878785/1125550745312313354

**Comment by dhiyaa777** [comment]
guys i still get the crash in rpc3  BLES00891

**Comment by AniLeo** [comment]
Need a new log file for the crash

**Comment by digant73** [comment]
is that old bug still present?

**Comment by digant73** [comment]
@Jonathan44062, @Triticum0 can you please provide an update on this issue?

**Comment by Jonathan44062** [comment]
I don't have the game, I just left my comment because I saw the connection with Sonic Unleashed, but it's no longer there and it's stable. Ask @solarmystic, he's the one who has to confirm if Driver: San Francisco has been fixed.

**Comment by Triticum0** [comment]
@digant73 I only tested the demo, but nfxw was able to complete the demo 2 week ago so at least on demo version should be fixed https://forums.rpcs3.net/showthread.php?tid=206224

**Comment by digant73** [comment]
yes I did the same and no issue at all on current build 0.0.38-18327 while it was easily freezing on old builds (IIRC it crashed even on some builds on early 2024). Pretty much sure this issue is no longer present

**Comment by lostromb** [comment]
I played for an hour yesterday and encountered absolutely zero issues. It's possible it might make a difference if you use LLVM PPU because that's what I set in my game-specific config.

Windows 11 + Nvidia Maxwell + Ryzen 5000

[RPCS3.log.gz](https://github.com/user-attachments/files/24040370/RPCS3.log.gz)

I vote that this issue be closed.

**Comment by alexconsole04** [comment]
Played through most of Chapter 1 and Prologue with no freezing. It appears it has been fixed.

[RPCS3.log.gz](https://github.com/user-attachments/files/25217754/RPCS3.log.gz)

**Comment by digant73** [comment]
@main developers this issue has been confirmed as fixed by testers and users.  I also verified it was crashing on old builds (till 2024) and now it is absolutely stable. IMO the issue can be closed (as usual OP is also no more active)
